mods/ra/rules/palettes.yaml

```diff
@@ -1,7 +1,8 @@
 ^Palettes:
-	PlayerPaletteFromCurrentTileset:
+	PaletteFromFile@player:
 		Name: player
-		ShadowIndex: 3,4
```

Comment:
I think this change is responsible for https://github.com/OpenRA/OpenRA/issues/9989. Can you elaborate why this was necessary?